stdlib/char.gr

```diff
@@ -191,3 +191,71 @@ provide let toString = (char: Char) => {
 
   result
 }
+
+/**
+ * Checks if the first character is less than the second character by Unicode scalar value.
+ *
+ * @param x: The first value
+ * @param y: The second value
+ * @returns `true` if the first value is less than the second value or `false` otherwise
```

Comment:
```suggestion
 * @param x: The first character
 * @param y: The second character
 * @returns `true` if the first character is less than the second character or `false` otherwise
```
You changed the description but not the docs on args